src/main/java/uk/gov/companieshouse/confirmationstatementapi/interceptor/LoggingInterceptor.java

```diff
@@ -1,31 +1,25 @@
 package uk.gov.companieshouse.confirmationstatementapi.interceptor;
 
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 import org.slf4j.MDC;
 import org.springframework.stereotype.Component;
 import org.springframework.web.servlet.HandlerInterceptor;
 import org.springframework.web.servlet.HandlerMapping;
-import uk.gov.companieshouse.confirmationstatementapi.ConfirmationStatementApiApplication;
 
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+import static uk.gov.companieshouse.confirmationstatementapi.ConfirmationStatementApiApplication.LOGGER;
+
 @Component
 public class LoggingInterceptor implements HandlerInterceptor{
 
-    private static final Logger LOGGER = LoggerFactory.getLogger(LoggingInterceptor.class);
-
     @Override
     public boolean preHandle(HttpServletRequest request, HttpServletResponse response, Object handler) {
         Long startTime = System.currentTimeMillis();
         request.getSession().setAttribute("start-time", startTime);
 
         MDC.put("context", requestId(request));
-        MDC.put("namespace", ConfirmationStatementApiApplication.APP_NAME);
-        if (LOGGER.isInfoEnabled()) {
-            LOGGER.info("Start of request. Method: {} Path: {}", requestMethod(request), requestPath(request));
-        }
+        LOGGER.info(String.format("Start of request. Method: %s Path: %s", requestMethod(request), requestPath(request)));
```

Comment:
you should put the context into the logger method using LOGGER.infoContext and remove the MDC line